feat: add presentation for load balancing

xanvierb opened the pull request.

xanvierb committed:
feat: add presentation for load balancing

johnsterken commented on index.html:
```diff
@@ -16,6 +16,7 @@
         <ul>
             <li>An overview of all presentations in reverse chronological order</li>
             <li><a href="intellij">2022-08-03 - IntelliJ</a></li>
+            <li><a href="loadtesting">2022-08-17 - Load testing</a></li>
```
Deze moet boven IntelliJ komen. De lijst is bedoeld om de meest recente bovenaan te hebben

xanvierb committed:
chore: fix order of presentations folder names

Folder names are now prefixed with the date of the presentatie so
they are in order in your file system/ide.
